src/controllers/user.js

```diff
@@ -47,13 +47,21 @@ export const getById = async (req, res) => {
 
 export const getAll = async (req, res) => {
   // eslint-disable-next-line camelcase
-  const { first_name: firstName, cohort: inCohort } = req.query
+  const {
+    first_name: firstName,
+    cohort: inCohort,
+    cohort_id: cohortId
+  } = req.query
```

Comment:
Why  are you using```req.query``` instead of ```req.body```?

Reply:
Because this information is being sent in the url as a query string (after the "?").